check-services.php

```diff
@@ -33,19 +34,23 @@
 echo "Starting service polling run:\n\n";
 $polled_services = 0;
 
-$where = '';
-$params = [];
+$services = Device::select('devices.*', 'services.*', 'devices_attribs.attrib_value')
```

Comment:
This variable should be $query or similar to indicate that it is still being built.